api/envoy/extensions/http/cache/file_system_http_cache/v3/file_system_http_cache.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+package envoy.extensions.http.cache.file_system_http_cache.v3;
+
+import "envoy/extensions/common/async_files/v3/async_file_manager.proto";
+
+import "google/protobuf/wrappers.proto";
+
+import "xds/annotations/v3/status.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.http.cache.file_system_http_cache.v3";
+option java_outer_classname = "FileSystemHttpCacheProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/http/cache/file_system_http_cache/v3;file_system_http_cachev3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+option (xds.annotations.v3.file_status).work_in_progress = true;
+
+// [#protodoc-title: FileSystemHttpCacheConfig]
+// [#extension: envoy.extensions.http.cache.file_system_http_cache]
+
+// Configuration for a cache implementation that caches in the local file system.
+//
+// By default this cache uses a least-recently-used eviction strategy.
+//
+// Used in the ``typed_config`` field of ``envoy.extensions.filters.http.cache.v3.CacheConfig``.
+// [#next-free-field: 7]
+message FileSystemHttpCacheConfig {
+  // Configuration of a manager for how the file system is used asynchronously.
+  common.async_files.v3.AsyncFileManagerConfig manager_config = 1
+      [(validate.rules).message = {required: true}];
+
+  // Path at which the cache files will be stored.
+  //
+  // This also doubles as the unique identifier for a cache, so a cache can be shared
+  // between different routes, or separate paths can be used to specify separate caches.
+  //
+  // If the same ``cache_path`` is used in more than one ``CacheConfig``, the rest of the
+  // ``FileSystemHttpCacheConfig`` must also match, and will refer to the same cache
+  // instance.
```

Comment:
Is this validated and config rejected if this is not the case?

Reply:
Yes. Should that be explained in the comment, or are you just checking that my usage of 'must' is correct? :)